packages/node/src/types.ts

```diff
@@ -85,6 +85,15 @@ export interface BaseNodeOptions {
    */
   maxSpanWaitDuration?: number;
 
+  /**
+   * Whether to register ESM loader hooks to automatically instrument libraries.
+   * This is necessary to auto instrument libraries that are loaded via ESM imports, but might it can cause issues
+   * with certain libraries.
+   *
+   * Defaults to `true`.
+   */
```

Comment:
```suggestion
  /**
   * Whether to register ESM loader hooks to automatically instrument libraries.
   * This is necessary to auto instrument libraries that are loaded via ESM imports, but might it can cause issues
   * with certain libraries. If you run into problems running your app with this enabled,
   * please raise an issue in https://github.com/getsentry/sentry-javascript.
   *
   * Defaults to `true`.
   */
```

should we be very explicit here?